src/handlers/block_action.handler.ts

```diff
@@ -131,11 +131,18 @@ function handleCompleteAction(payload: any, response: express.Response, actionId
 
     const workspaceId = payload.team.id;
     const channelId = payload.channel.id;
+    const username = payload.user.name;
 
     const queryText = "DELETE FROM action_items WHERE id=$1";
     const queryValues = [actionId];
 
-    client.query(queryText, queryValues)
+    let itemDescription = "an item";
+
+    client.query('select * from action_items where id=$1', [actionId])
+        .then(res => {
+            itemDescription = res.rows[0].description;
+            return client.query(queryText, queryValues);
+        })
```

Comment:
If we used `async/await` here instead of `.then`, it might look a little nicer

Reply:
good idea. Lemme try that out...

Reply:
looks better to me 🤷‍♂️